content/reference/admin/users/index.md

```diff
@@ -9,19 +9,21 @@ weight: 20010
 
 ## Managing Users
 
-To access the Users administration, click on the {{< icon user-cog >}} icon in the navigation bar.
+To access the Users administration, click on the {{< icon user >}} icon in the navigation bar.
 
 {{< img src="users-table.png" alt="Users table" >}}
 
-To create a user, click on the **Create** button and fill the user information:
+To invite a user to your organization, click on the **Create** button and fill the user information:
 
-- **Username**: must match user Github username in order to connect and *cannot be updated*.
-- **Firstname** | **Lastname** | **Email**: explicit
-- **Roles**: either choose a global role, or select none and specify each role by team according to the permissions you wan't to grant.
+- **Github Username**: must match user Github username in order to connect and *cannot be updated*.
+- **Firstname** | **Lastname** | **Email address**: explicit
```

Comment:
```suggestion
- **Firstname** | **Lastname** | **Email address**: explicit. The user will be able to change these once they are connected.
```
(maybe not needed, but since the other fields are either not modifiable at all, or not by the user themselves...)